lib/posix/options/Kconfig.pipe

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024, Bruce Rosier
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menu "POSIX Pipe Options"
+
+config POSIX_PIPE
+	bool"POSIX pipe support"
+	default y
+	depends on POSIX_API
```

Comment:
This should depend on I guess `FDTABLE` and maybe some other options. `Kconfig.net` might have some hints.

```suggestion
```

Also, below this, please add 

```
if POSIX_PIPE
...
endif
```